process/cg-transition.html

```diff
@@ -106,50 +106,26 @@ <h3>
     <h3>
       Are IPR commitments in place?
     </h3>
-    <p>
-      Community Groups have a "two-step IPR policy" so that people make
-      lightweight commitments when work starts and more comprehensive
-      commitments once work is mature.
-    </p>
-    <ul>
-      <li>Initially, a participant makes certain commitments regarding their
-      own Contribution under the <a href=
-      "https://www.w3.org/community/about/process/cla/">W3C Community
-      Contributor Licensing Agreement (CLA)</a>.
-      </li>
-      <li>Later, a participant can make a voluntary commitment regarding the
-      entire specification under the <a href=
-      "https://www.w3.org/community/about/agreements/final/">W3C Community
-      Final Specification Agreement (FSA)</a>.
-      </li>
-    </ul>
-    <p>
-      By comparison, under the W3C Patent Policy for Working Groups,
-      participants make full specification commitments upon joining the group.
-    </p>
+    <p>When a Working Group plans to take up a Community Group specification, it is important that there be "IPR safety" around the specification managed by the Working Group.</p>
```

Comment:
```suggestion
    <p>
      When a Working Group plans to take up a Community Group specification,
      it is important that the Working Group manage "IPR safety" around the
      specification.
    </p>
```

Reply:
Hi @TallTed, I prefer the original text here. Some of the steps might not involve the Working Group, which is why I had chosen the passive voice.

Reply:
I see what you mean. Given that, I think it needs a different rephrasing, as my suggestion above was based on what I read the original as saying. Do you want a PR from me for this, or will you apply it?

```suggestion
    <p>
      When a Working Group plans to take up a Community Group specification,
      it is important that there be "IPR safety" around the resulting Working
      Group specification.
    </p>```

Reply:
I've made the change, thanks!